tests/subsys/fs/fcb/src/main.c

```diff
@@ -13,7 +13,9 @@
 #include <zephyr/drivers/flash.h>
 #include <zephyr/device.h>
 
-struct fcb test_fcb;
+struct fcb test_fcb = {0};
+struct fcb test_fcb_crc_disabled = { .f_crc_disable = 1 };
```

Comment:
provide flag definition in fcb.h and macro to set the flag. Usually we do not use the bitfields but directly address bits in flag variable.